x/emissions/types/inference_request.go

```diff
@@ -45,8 +45,8 @@ func CreateNewInferenceRequestFromListItem(sender string, item *RequestInference
 		Cadence:              item.Cadence,
 		MaxPricePerInference: item.MaxPricePerInference,
 		BidAmount:            item.BidAmount,
-		TimestampValidUntil:  item.TimestampValidUntil,
-		LastChecked:          0,
+		BlockValidUntil:      item.BlockValidUntil,
+		BlockLastChecked:     0,
```

Comment:
this is going to break rewards.sh in the integration folder fyi, but I agree with the change

Reply:
fixed it